JASP-Engine/JASP/R/LScommon.R

```diff
@@ -1629,14 +1629,14 @@ hdi.density    <- function(object, credMass=0.95, allowSplit=FALSE, ...) {
              gettextf("Model - for specifying models that will be used for estimation. You can specify the model name, the prior distribution for the parameter %s, and parameters that define the distribution.", parameter_info),
              gettextf("Hypothesis - for setting hypotheses that will be used for testing. You can specify the hypothesis name, the prior probability, the prior distribution for the parameter %s, and parameters that define the distribution.", parameter_info)),
       ifelse(estimation,
-             gettext("Inference - for visualizing the specified models. The individual options show prior, posterior, and prior and posterior distributions. In addition, the multiple models can be shown in one figure (All), with a depth effect (Stacked), or in individual figures (Individual)."),
-             gettext("Inference - for visualizing the specified hypothesis. The individual options show prior distributions, prior predictive distributions, posterior distributions, prior and posterior distributions, and predictive accuracy. In addition, the multiple hypotheses can be shown when considered alone (Conditional), after multiplication by the prior (Joint), and combined together (Marginal)." )),
+             gettext("Inference - for drawing conclusions from the specified models. The options include visualizing prior, posterior, and prior and posterior distributions. In addition, the multiple models can be shown in one figure (All), with a depth effect (Stacked), or in individual figures (Individual)."),
+             gettext("Inference - for drawing conclusions based on the specified hypothesis. The options include visualizing prior distributions, prior predictive distributions, posterior distributions, prior and posterior distributions, and predictive accuracy. In addition, the multiple hypotheses can be shown when considered alone (Conditional), after multiplication by the prior (Joint), and combined together (Marginal)." )),
       ifelse(estimation,
-             gettext("Sequential analysis - for displaying the Bayesian estimation updating with additional data (available only with non-aggregated data). The individual options show updating of the point estimate, specified interval, and the resulting distributions."),
-             gettext("Sequential analysis - for displaying Bayesian evidence accumulation as data come in (available only with non-aggregated data).")),
+             gettext("Sequential analysis - for displaying the results of Bayesian estimation sequentially, one observation at the time (available only with non-aggregated data). The individual options show updating of the point estimate, specified interval, and the resulting distributions."),
```

Comment:
See https://github.com/jasp-stats/jasp-test-release/issues/965

Reply:
Thanks, I forgot to change the second part of the sentence.